### PR TITLE
Use go modules

### PR DESCRIPTION
**Pull request by obbardc**

### CODE REVIEW

**Comment by sjoerdsimons** [comment]
@obbardc draft but you'd like a rview ? that's confusing :)

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module github.com/go-debos/debos
+
+go 1.15
+
+require (
+	github.com/docker/go-units v0.4.0
+	github.com/go-debos/fakemachine v0.0.0-20220412175549-8719c1b0b5bb
```
Comment (sjoerdsimons) [inline go.mod:7]: time to properly tag a fakemachine release?

**Comment by obbardc** [comment]
> @obbardc draft but you'd like a rview ? that's confusing :)

Just to see early if there is anything glaringly wrong :-)

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 2
```
Comment (sjoerdsimons) [inline .github/dependabot.yml:1]: Can you split out this commit in a seperate MR

File: go.mod [hunk 3]
```diff
@@ -0,0 +1,20 @@
+module github.com/go-debos/debos
```
Comment (sjoerdsimons) [inline go.mod:1]: This can be merge seperately from introducing better tsting so pleas split it out

File: go.mod (hunk 1 above)
Comment (obbardc) [reply go.mod:7]: not quite; but i think in april we will do so ;-)

**Pull request merged**